### PR TITLE
Add android as a prefix in every package

### PR DESCRIPTION
**Pull request by felipecsl**

### CODE REVIEW

File: apollo-api/src/main/java/com/apollographql/android/api/graphql/Response.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+import java.util.List;
+
+import javax.annotation.Nullable;
+
+/** Represents either a successful or failed response received from the GraphQL server. */
+public class Response<T extends com.apollographql.android.api.graphql.Operation.Data> {
+  @Nullable private final T data;
+  @Nullable private final List<com.apollographql.android.api.graphql.Error> errors;
+
+  public Response(@Nullable T data, @Nullable List<com.apollographql.android.api.graphql.Error> errors) {
```
Comment (digitalbuddha) [inline apollo-api/src/main/java/com/apollographql/android/api/graphql/Response.java]: whats up with this class being recreated both prs?  Also is the fully qualified name in `List<>` what you expected?

File: apollo-compiler/src/main/kotlin/com/apollographql/android/compiler/OperationTypeSpecBuilder.kt [hunk 2]
```diff
@@ -82,7 +82,7 @@ class OperationTypeSpecBuilder(
         .addAnnotation(Annotations.OVERRIDE)
         .addModifiers(Modifier.PUBLIC)
         .returns(queryFieldClassName)
-        .addStatement("return $VARIABLES_FIELD_NAME")
+        .addStatement("return ${VARIABLES_FIELD_NAME}")
```
Comment (digitalbuddha) [inline apollo-compiler/src/main/kotlin/com/apollographql/android/compiler/OperationTypeSpecBuilder.kt]: could you explain this change?

File: apollo-compiler/src/main/kotlin/com/apollographql/android/compiler/ir/InlineFragment.kt [hunk 3]
```diff
@@ -33,7 +33,7 @@ data class InlineFragment(
           .addModifiers(Modifier.PRIVATE)
           .build()
 
-  private fun interfaceName() = "$INTERFACE_PREFIX${typeCondition.capitalize()}"
+  private fun interfaceName() = "${INTERFACE_PREFIX}${typeCondition.capitalize()}"
```
Comment (digitalbuddha) [inline apollo-compiler/src/main/kotlin/com/apollographql/android/compiler/ir/InlineFragment.kt]: same as before, was this just in error before?

File: apollo-compiler/src/main/kotlin/com/apollographql/android/compiler/ir/graphql/Type.kt [hunk 4]
```diff
@@ -25,30 +25,30 @@ sealed class Type(val isOptional: kotlin.Boolean) {
 
   companion object {
     private val GRAPHQLTYPE_TO_JAVA_TYPE = mapOf(
-        Type.String::class.java to ClassNames.STRING,
-        Type.Id::class.java to ClassNames.STRING,
-        Type.Int::class.java to TypeName.INT,
-        Type.Float::class.java to TypeName.DOUBLE,
-        Type.Boolean::class.java to TypeName.BOOLEAN)
+        String::class.java to ClassNames.STRING,
```
Comment (digitalbuddha) [inline apollo-compiler/src/main/kotlin/com/apollographql/android/compiler/ir/graphql/Type.kt]: sanity check that you wanted to remove `Type.` here

File: apollo-converters/pojo/src/main/java/com/apollographql/android/converter/pojo/ApolloConverterFactory.java [hunk 5]
```diff
@@ -11,9 +11,9 @@
 import retrofit2.Converter;
 import retrofit2.Retrofit;
 
+/** TODO */
 public class ApolloConverterFactory extends Converter.Factory {
-  @Override
-  public Converter<ResponseBody, Response<? extends Operation.Data>> responseBodyConverter(Type type,
+  @Override public Converter<ResponseBody, Response<? extends Operation.Data>> responseBodyConverter(Type type,
```
Comment (digitalbuddha) [inline apollo-converters/pojo/src/main/java/com/apollographql/android/converter/pojo/ApolloConverterFactory.java]: why the style change?

File: apollo-compiler/src/main/kotlin/com/apollographql/android/compiler/OperationTypeSpecBuilder.kt (hunk 2 above)
Comment (felipecsl) [inline apollo-compiler/src/main/kotlin/com/apollographql/android/compiler/OperationTypeSpecBuilder.kt]: This is a quick fix suggested by the IDE, the curly braces are not needed here. It's totally unrelated to the PR, so I'm happy to revert this if you dont like it

File: apollo-compiler/src/main/kotlin/com/apollographql/android/compiler/ir/InlineFragment.kt (hunk 3 above)
Comment (felipecsl) [inline apollo-compiler/src/main/kotlin/com/apollographql/android/compiler/ir/InlineFragment.kt]: Same thing

File: apollo-compiler/src/main/kotlin/com/apollographql/android/compiler/OperationTypeSpecBuilder.kt (hunk 2 above)
Comment (digitalbuddha) [inline apollo-compiler/src/main/kotlin/com/apollographql/android/compiler/OperationTypeSpecBuilder.kt]: keep it..was just wondering if something was broken or just different syntax

File: apollo-api/src/main/java/com/apollographql/android/api/graphql/Response.java (hunk 1 above)
Comment (felipecsl) [inline apollo-api/src/main/java/com/apollographql/android/api/graphql/Response.java]: Maybe not, I'll recheck!

File: apollo-compiler/src/main/kotlin/com/apollographql/android/compiler/ir/graphql/Type.kt (hunk 4 above)
Comment (felipecsl) [inline apollo-compiler/src/main/kotlin/com/apollographql/android/compiler/ir/graphql/Type.kt]: yeah this looks correct

File: apollo-converters/pojo/src/main/java/com/apollographql/android/converter/pojo/ApolloConverterFactory.java (hunk 5 above)
Comment (felipecsl) [inline apollo-converters/pojo/src/main/java/com/apollographql/android/converter/pojo/ApolloConverterFactory.java]: Unrelated.
